src/rgw/rgw_sync_module_es.cc

```diff
@@ -304,19 +319,30 @@ struct es_obj_metadata {
             }
           }
         }
-      } else if (name == "x-amz-tagging") {
-        auto tags_bl = val.begin();
-        decode(obj_tags, tags_bl);
-      } else if (name == "compression") {
+      } else if (attr_name == RGW_ATTR_TAGS) {
+        try {
+          auto tags_bl = val.begin();
```

Comment:
@theanalyst Need review. the bufferlist cbegin() is not available in mimic.

Reply:
This is ok

Reply:
Please see comments on const char* for initializer_list